sdk/Cargo.toml

```diff
@@ -82,14 +75,15 @@ features = [
   "autonat"
 ]
 
-[target.'cfg(target_family = "wasm")'.dependencies.libp2p]
-workspace = true
-default-features = false
+# WASM-only deps
+[target.'cfg(all(any(target_arch = "wasm32", target_arch = "wasm64"), target_os = "unknown"))'.dependencies]
```

Comment:
Do we still need this block though?

Reply:
The SDK should just need `std` feature afaict?

Reply:
We'll still need a `wasm` feature. The `getrandom` js feature needs to be enabled for wasm32-unknown-unknown, but I can move that to the feature instead of having it automatically enabled for the target.

Reply:
I'm not entirely convinced we need getrandom, but I might be mistaken. My frame of reference is arkworks and substrate packages.

Reply:
We don't actually use it directly, we're just required to explicitly enable the `js` feature to get it to build since `rand_core` is used in key generation.